os/arch/arm/src/stm32l4/stm32l4_idle.c

```diff
@@ -224,7 +224,7 @@ static void up_idlepm(void)
 		/* Enable tick interrupts */
 		enable_ticks();
 
-		oldstate = PM_IDLE;
+		//oldstate = PM_IDLE;	/* Re visit */
```

Comment:
This change can be added to a different commit.
Same for above changes added to this commit in stm32l4_idle.c

Reply:
Done